.eslintrc.json

```diff
@@ -67,12 +68,13 @@
       "publicOnly": true,
       "enableFixer": false
     }],
-    "jsdoc/require-param": ["error"],
-    "jsdoc/require-param-type": ["error"],
-    "jsdoc/require-returns": ["error"],
-    "jsdoc/require-returns-type": ["error"],
+    "jsdoc/require-param": "error",
+    "jsdoc/require-param-type": "error",
+    "jsdoc/require-returns": "error",
+    "jsdoc/require-returns-type": "error",
     "no-unused-vars": "off",
     "no-var": "error",
-    "prettier/prettier": "error"
+    "prettier/prettier": "error",
+    "security/detect-object-injection": "off"
```

Comment:
I'm happy to be argued with on this, I don't think it provides us a lot of value.